RubeesEat/Model/EntryLine.cs

```diff
@@ -8,6 +8,11 @@ public class EntryLine
     public EntryLine(Person person, decimal amount)
     {
         ArgumentNullException.ThrowIfNull(person);
+
+        if (!person.IsActive)
+        {
+            throw new ArgumentException("Person is not active!");
+        }
```

Comment:
Why throw Exception? We have bills where the entryline is from an inactive user, e.g. if you show all bills, there will be many old bills with inactive users. If we throw an exception, the website will just stop working.